app/frontend/src/components/ImageInput.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export interface ImageInputProps {
+    id?: string;
+    name?: string;
+    onChange: (file: File) => void;
+}
+
+export const ImageInput: React.FC<ImageInputProps> = ({id, name, children, onChange}) => {
+    const inputId = id + '-file-input';
+    return (
+        <React.Fragment>
+            <input
+                accept="image/*"
+                style={{display: 'none'}}
```

Comment:
use `className`
(Hmm if this is the only style in the file, maybe you don't have to).

Reply:
In order to make the code organized, we should just keep it consistent and just use `className`.

Reply:
I'll change it and remember to use className in the future aswell